src/kernl/implementations/attention_skinny.py

```diff
@@ -147,196 +147,200 @@ def _fwd_part_1(
     )
 
     # pointers to blocks in Q, K, V
-    ptrs_q = Q + offs_q
-    ptrs_k = K + offs_k
-    ptrs_v = V + offs_v
+    ptrs_q = q_ptr + offs_q
```

Comment:
`q_ptrs`, `k_ptrs` and `v_ptrs` ?

Reply:
![image](https://user-images.githubusercontent.com/1029874/211839560-4180f559-bcde-4dc5-96ae-8580ef84245f.png)

Reply:
done